docs/platform/manager/third-party-authentication/index.md

```diff
@@ -0,0 +1,60 @@
+---
+author:
+  name: Linode Community
+  email: docs@linode.com
+description: 'This guide shows you how to enable Third Party Authentication (TPA) for your Linode account, allowing you to log in to Linode Cloud Manager with third party credentials.'
+og_description: 'This guide shows you how to enable Third Party Authentication (TPA) for your Linode account, allowing you to log in to Linode Cloud Manager with third party credentials.'
+keywords: ['tpa','third party authentication','github','account','password']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-04-22
+modified_by:
+  name: Linode
+title: "How to Enable Third Party Authentication on Your Linode Account"
+h1_title: "Enabling Third Party Authentication on Your Linode Account"
+---
+
+Linode Cloud Manager supports Third-Party Authentication (TPA). This allows you to log in to the Linode Cloud Manager with another provider's login credentials.
+
+## Enabling Third Party Authentication
+
```

Comment:
We should note to users that this will disable TFA devices for Cloud and that they should utilize TFA with their provider.

It also disables their LISH password for accessing a specific LISH SSH gateway, but they can still utilize key authentication.

Reply:
I just realized we didn't include this in the manager changes that are up for review. I've made a note to them that we should include notifications to the user about this as well.